sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ResourceGroupOperations.cs

```diff
@@ -133,7 +130,7 @@ public virtual ArmOperation StartDelete(CancellationToken cancellationToken = de
 
             try
             {
-                return new PhVoidArmOperation(Operations.StartDelete(Id.Name, cancellationToken));
+                return new PhVoidArmOperation(RestClient.Delete(Id.Name, cancellationToken));
```

Comment:
should return the ResourceGroupDeleteOperation, this is true for all LRO methods they need to return the concrete LRO class